### PR TITLE
Formatting suggestions / Include async/await approach for promise handling

### PR DESCRIPTION
**Pull request by francocorreasosa**
## Description
Hey 👋 I saw this docs were a little but out of date with the current state of JS and decided to contribute fixing some things I found plus making formatting suggestions in both JS and Markdown (following _Prettier_ standards)

### Checklist:
- [x] This request modifies code snippets. 
- [x] The contribution aligns with the information stated in the repository wiki.
- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [x] Requires translations.

### CODE REVIEW

**Comment by lamaralmp** [comment]
Gracias por tu aporte.
Vamos a revisarlo

**Comment by lamaralmp** [comment]
Traducción realizada. En espera de la aprobación del equipo SDK

File: plugins_sdks/sdks/official/nodejs.es.md [hunk 1]
```diff
@@ -1,84 +1,102 @@
...
+    console.log(accessToken);
+  }
 });
 ```
 
 Para utilizar callbacks, simplemente pasa una función como último parámetro.
 
+### Soporte a `async` / `await`
+
+Cada metodo soporta el uso de `await`.
```
Comment (mminestrelli) [inline plugins_sdks/sdks/official/nodejs.es.md]:
```suggestion
Cada método soporta el uso de `await`.
```

File: plugins_sdks/sdks/official/nodejs.es.md [hunk 2]
```diff
@@ -88,70 +106,85 @@ _Configuración de credenciales inválidas:_
...
 
 ```javascript
 mercadopago.configure({
-    sandbox: true
+  sandbox: true
 });
 ```
 
-> Debes proporcionar un metodo de autentificación (client_id & client_secret or access_token).
+> Debes proporcionar un metodo de autentificación (`client_id` & `client_secret` or `access_token`).
```
Comment (mminestrelli) [inline plugins_sdks/sdks/official/nodejs.es.md]:
```suggestion
> Debes proporcionar un método de autenticación (`client_id` & `client_secret` o `access_token`).
```

File: plugins_sdks/sdks/official/nodejs.es.md [hunk 3]
```diff
@@ -88,70 +106,85 @@ _Configuración de credenciales inválidas:_
...
 ```javascript
 mercadopago.configure({
-    client_secret: 'CLIENT_SECRET'
+  client_secret: "CLIENT_SECRET"
 });
 ```
 
-> No se puede cambiar el client_id o client_secret porque ya estan configuradas.
-
+> No se puede cambiar el `client_id` o `client_secret` porque ya estan configuradas.
```
Comment (mminestrelli) [inline plugins_sdks/sdks/official/nodejs.es.md]:
```suggestion
> No se puede cambiar el `client_id` o `client_secret` porque ya están configuradas.
```

File: plugins_sdks/sdks/official/nodejs.es.md [hunk 4]
```diff
@@ -88,70 +106,85 @@ _Configuración de credenciales inválidas:_
...
 
 ## Modo Sandbox
 
-Si vas a utilizar nuestra aplicación en modo Sandbox, debes proporcionar el access_token.
+Si vas a utilizar nuestra aplicación en modo Sandbox, debes proporcionar el `access_token`.
 
-> Tenga en cuenta que si proporciona el client_id y client_secret, la SDK generará un access_token productivo.
+> Tenga en cuenta que si proporciona el `client_id` y `client_secret`, la SDK generará un access_token productivo.
```
Comment (mminestrelli) [inline plugins_sdks/sdks/official/nodejs.es.md]:
```suggestion
> Ten en cuenta que si proporcionas el `client_id` y `client_secret`, la SDK generará un access_token productivo.
```

**Comment by mminestrelli** [comment]
@francocorreasosa gracias por tu colaboración, tus cambios se incluyen en #622 .

**Pull request closed without merging**